lib/features/modeling/cmd/ReconnectConnectionHandler.js

```diff
@@ -49,21 +49,19 @@ ReconnectConnectionHandler.prototype.postExecute = function(context) {
     return;
   }
 
-  var layoutConnectionHints = {};
-
-  if (newSource && (!newTarget || hints.docking === 'source')) {
-    layoutConnectionHints.connectionStart = getDocking(isArray(dockingOrPoints) ? dockingOrPoints[ 0 ] : dockingOrPoints);
```

Comment:
We monkey patch `layoutConnectionHints` here. Is this intentional?

Reply:
Only if the hint doesn't exist. We need to be able to pass the `connectionStart` and `connectionEnd` hints when reconnecting in dmn-js through moving a bendpoint. The DMN-specific snapping provides these hints.

Reply:
Understood. But why do we monkey patch something that is provided from outside, i.e. the `hints` object?

Reply:
Actually, the hints are never passed in the way I expected them to be. 💩

Reply:
Is this fixed / resolved?

Reply:
Fixed.